checkout/frames-android
# feature/PIMOB-2262_Artifact_generation_and_download_via_GHA

## Issue

PIMOB-2262

## Proposed changes

Simplification of the general flows; loads of jobs (unit test, assemble and lint) were always enabled for all the flows, so I have removed the redundant flag.
While doing the assemble task, I have introduced a new boolean flag to:
- create both debug and release builds
- provide the output of those available on github action

<img width="1448" alt="Screenshot 2023-11-27 at 17 19 07" src="https://github.com/checkout/frames-android/assets/123394131/27ae8285-5c0b-499e-b2fb-12615f53c8da">

## Test Steps

1. Do a change on checkout module
2. verify that all the job will trigger
3. Verify that only app and example-app flow will have artifacts to download

## Checklist
